userguide/tutorials/metrics-newrelic.adoc

```diff
@@ -82,38 +82,55 @@ installing the *New Relic Agent* in the system producing the data.
 
 image::metrics-newrelic-installation.png[align=center]
 
-5. In the box containing the install command, click *Copy Command*. The command will be copied to your clipboard.
+[start=5]
+. In the box containing the install command, click *Copy Command*. The command will be copied to your clipboard.
 
 [NOTE]
 This command includes the API key assigned to you by
 New Relic. This authorizes the agent to do its job. This is your *user* API Key, and should not be confused with the key you will use in the next step.
 
 To install the agent: 
 
-1. Login to your EC2 instance. 
-2. Paste the saved install command to the Ubuntu command line. 
-3. Press Enter. The installation should begin.
+[start=6]
+.. Login to your EC2 instance. 
+.. Paste the saved install command to the Ubuntu command line. 
+.. Press Enter. The installation should begin.
 
-You will be presented with a list of options to install, as shown below. An additional option, Infrastructure Agent, is always installed. Press Enter to select all of the options.
+You will be presented with a list of options to install, as shown below. An additional option, Infrastructure Agent, is always installed.
+
+[start=7]
+. Press Enter to select all of the options.
 
```

Comment:
Again numbering does not look right. After `5`, the next number is `7`, so `6` is missing.

Reply:
fixed

Reply:
fixed